server/src/services/email.ts

```diff
@@ -0,0 +1,110 @@
+import Koa from "koa";
+import { IConfig } from "../config";
+import getEmailer, { IEmailer } from "../email";
+
+export interface IEmailService {
+  sendMessage(
+    ctx: Koa.Context,
+    recipient: string,
+    subject: string,
+    message: string
+  ): Promise<void>;
+
+  sendApproval(
+    ctx: Koa.Context,
+    recipient: string,
+    formTitle: string
+  ): Promise<void>;
+
+  sendRejection(
+    ctx: Koa.Context,
+    recipient: string,
+    formTitle: string
+  ): Promise<void>;
+
+  sendInterviewRequest(
+    ctx: Koa.Context,
+    recipient: string,
+    formTitle: string,
+    scheduleURL: string
+  ): Promise<void>;
+
+  sendInterviewConfirmation(
+    ctx: Koa.Context,
+    recipient: string,
+    formTitle: string,
+    interviewSlot: string
+  ): Promise<void>;
+}
+
+class ConcreteEmailService implements IEmailService {
+  private config: IConfig;
+  constructor(config: IConfig) {
+    this.config = config;
+    this.emailer = getEmailer(this.config);
```

Comment:
nit: I think the DI way to do this would be to just pass in the Emailer directly, so that you could mock it out as easily as possible

Reply:
Check recent commit -- like that?

Reply:
Yeah that's exactly what I was thinking

Reply:
Dope I'll merge